src/components/results.js

```diff
@@ -1,88 +1,96 @@
-import React, { Component } from 'react';
+import React, {Component} from 'react';
 import axios from 'axios';
 
 import api from './api';
 
 export default class Results extends Component {
-	constructor(props) {
-		super(props)
-		this.state = {
-			data: {},
-			canVote: null,
-		};
-	}
+    constructor(props) {
+        super(props)
+        this.state = {
+            data: {},
+            canVote: null,
+        };
+    }
 
-	componentDidMount(props) {
-				this.user_can_vote();
-	}
+    componentDidMount(props) {
+        this.user_can_vote();
+    }
 
 
-     //Method to determine if you can vote
+    //Method to determine if you can vote
 
     user_can_vote() {
```

Comment:
@DahirIlhan let's follow [javascript naming conventions](https://github.com/airbnb/javascript#naming-conventions) for this and use camel case. underscores are usually used in Ruby